### ISSUE TITLE
itunes

### ISSUE THREAD

**Issue by despoinamusica39** [qkd888/qkdev-iTunes-Search-API-App#1]
hey you delete your project?why?

**Comment by qkd888** [comment]
no, its back

**Comment by qkd888** [comment]
I had some issues with deployment on heroku, but its sorted now.

**Comment by qkd888** [comment]
And I updated the color palette

**Comment by despoinamusica39** [comment]
aaaaaa okkkkkkkkkk...I Thought that you are ungry with me!!!

**Comment by despoinamusica39** [comment]
pleaseeeeeeeeeee increase unlimited results with offset 0

**Comment by qkd888** [comment]
The results are limited to the search query, so unfortunately not. It work smooth as it is now.

**Comment by despoinamusica39** [comment]
no,you can...check this please!!!!!!!!!!!!!! https://stackoverflow.com/questions/44177089/itunes-search-api-page-number-for-the-query


I found a way to query more records. There is a parameter called offset, which is not mentioned in API documentation.

https://itunes.apple.com/search?term=a&offset=25&limit=25
